backend/prompts/prompting_test.py

```diff
@@ -0,0 +1,21 @@
+from prompting import PromptEngine
+
+def test_mistral_prompt_engine_creation():
+    try:
+        PromptEngine("mistral")
+    except Exception:
+        raise
+
+def test_load_task_step_template():
+    engine = PromptEngine("mistral")
+    try:
+        engine.load_prompt("task-step")
+    except Exception:
+        raise
+
+def test_load_agent_selection_format_template():
+    engine = PromptEngine("mistral")
+    try:
+        engine.load_prompt("agent-selection-format")
```

Comment:
is there an assertion we can do here and for the other tests? (Forgive me if I'm missing it)

Reply:
I've added full string assertions